### ISSUE TITLE
Module build failed Error: PostCSS 

### ISSUE THREAD

**Issue by ivansky** [teambit/bit#4110]
### Describe the bug

After upgrade to 0.0.355 `bit tag --skip-tests` starts throwing PostCSS errors, when there is not even css at all.

### Steps to Reproduce

1. bit tag --skip-tests

### Expected Behavior

At least postcss should not be used when there is no css

### Screenshots, exceptions and logs
```
Module build failed (from ../../.bvm/versions/0.0.355/bit-0.0.355/node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ../../.bvm/versions/0.0.355/bit-0.0.355/node_modules/postcss-loader/dist/cjs.js):
Error: PostCSS plugin postcss-flexbugs-fixes requires PostCSS 8.
Migration guide for end-users:
https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users
    at Processor.normalize (/Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/postcss/lib/processor.js:153:15)
    at new Processor (/Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/postcss/lib/processor.js:56:25)
    at postcss (/Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/postcss/lib/postcss.js:55:10)
    at Object.loader (/Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/postcss-loader/dist/index.js:87:20)
    at processResult (/Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/webpack/lib/NormalModule.js:675:19)
    at /Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/webpack/lib/NormalModule.js:777:5
    at /Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/loader-runner/lib/LoaderRunner.js:399:11
    at /Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/loader-runner/lib/LoaderRunner.js:251:18
    at context.callback (/Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/loader-runner/lib/LoaderRunner.js:124:13)
    at Object.loader (/Users/ivansky/.bvm/versions/0.0.355/bit-0.0.355/node_modules/postcss-loader/dist/index.js:96:7)
...
Found 160 errors in 2 components
✖ Total 21 tasks. 17 succeeded. 4 failed. 0 skipped. Total errors: 480.
```

### Specifications

- Bit version: 0.0.355
- Node version: 14.16
- yarn version: 1.22.5
- Platform: darwin x64

**Comment by GiladShoham** [comment]
I'm working on this, thanks for reporting

**Comment by GiladShoham** [comment]
should be resolved by https://github.com/teambit/bit/commit/e2b4e1f1fcdd281b8684b3db2002f06274d40a45

**Comment by GiladShoham** [comment]
fixed in v0.0.358
